frontend/src/components/recommended-recipes/RecommendedRecipes.js

```diff
@@ -67,7 +67,7 @@ class RecommendedRecipes extends Component {
     }
 
     if (this.state.isLoading) {
-      return loading("Scanning recipes ...");
+      return loading("Scanning recipes...", /** withBackground*/ true);
```

Comment:
what does the /** withBackground*/ do

Reply:
See function loading below. If no text or withBackground are provided, the function will use the default values which are "Loading ..." and display loading without any background. 

For most requests to the backend, it doesn't take that long to get the response back, so for pages such as Favourites or History that have white background, the loading with a background might look like a glitch. That is why I created this withBackground variable with default value false as most pages have white background.

Whereas for scanning recipes, it takes quite a while to load the recipes from the backend so a background picture looks nicer.
```
export function loading(text = "Loading...", withBackground = false) {
  const background = withBackground ? getBackground() : "";
  return (
    <div>
      {background}
      <div className="spinner-div">
        <div className="spinner-border" role="status">
          <span className="sr-only">Loading...</span>
        </div>
        <h3>{text}</h3>
      </div>
    </div>
  );
}
```

Reply:
One thought here: Make sure to test your website in dark mode or provide a dark mode theme. Can be done after the main presentation.